metal/resource_metal_reserved_ip_block_test.go

```diff
@@ -230,15 +230,15 @@ resource "metal_project" "foobar" {
 
 resource "metal_reserved_ip_block" "test" {
 	project_id  = metal_project.foobar.id
-	facility    = "ny5"
+	facility    = "ewr1"
```

Comment:
We would want to go from ewr1 to ny5, not reverse.

Reply:
metros and facilities have been replaces with da/da11